components/local-app/cmd/workspace-ssh.go

```diff
@@ -15,7 +15,16 @@ var dryRun bool
 var workspaceSSHCmd = &cobra.Command{
 	Use:   "ssh <workspace-id>",
 	Short: "Connects to a workspace via SSH",
-	Args:  cobra.ExactArgs(1),
+	Args:  cobra.MinimumNArgs(1),
+	Example: `  # connect to workspace with current terminal session
+  $ gitpod workspace ssh <workspace-id>
+
+  # Execute simple command in the workspace
+  $ gitpod workspace ssh <workspace-id> -- ls -la
```

Comment:
Should not it already give all SSH features? basically it is a shortcut for constructing  proper ssh command?

Reply:
i.e. i should be able to things like

`gitpod workspace ssh <wid> -- -L 3336:db001.host:3306 user@pub001.host`?

Reply:
I thought we only support `--` feature, otherwise if we want to exec `ls`, we need to use: 

`gitpod ws ssh <wid> -- -- ls`

Reply:
addressed 

<img width="1523" alt="image" src="https://github.com/gitpod-io/gitpod/assets/20944364/498cd83b-6290-4015-88f7-4ab793637ec6">

Reply:
Case like `echo 'echo hello' | gitpod ws ssh gitpodio-gitpod-er1bcypgceo cat` is not covert, let's ignore it first ()